scripts/reannotate/prod45_idr0030_input.txt

```diff
@@ -0,0 +1 @@
+Screen:1801 idr0030-sero-yap/screenA openmicroscopy.org/mapr/phenotype
```

Comment:
This needs to be `Screen:1801 idr0030-sero-yap/screenA/idr0030-screenA openmicroscopy.org/mapr/phenotype`